Library/Formula/cv.rb

```diff
@@ -0,0 +1,15 @@
+class Cv < Formula
+  desc "Coreutils Viewer: Show progress for coreutils"
+  homepage "https://github.com/BestPig/cv"
```

Comment:
I've notices it. The official repo just merged BestPig's OSX support patch, but it's not included in latest release. So it still uses BestPig's fork.